html/main.html

```diff
@@ -4,8 +4,8 @@
 <%template "head" .%>
 </head>
 <body spreed-webrtc>
-<div id="background"></div>
-<div id="loader"><div><i class="fa fa-circle-o-notch fa-spin"></i><div class="loader-message"></div></div></div>
```

Comment:
Please keep the id for background and loader. If the style should be reuseable (questionable) then add an additional class.